pkgs/love2d/example/assets/modern_pong/libs/gamestateManager.lua

```
@@ -0,0 +1,91 @@
+    if self.currentState and self.currentState.enter then
+        self.currentState:enter()
+    end
+end
+
+function GameStateManager:mousemoved(x, y, ...)
+    if self.currentState and self.currentState.mousemoved then
+        self.currentState:mousemoved(x, y)
+    end
+end
+
+function GameStateManager:wheelmoved(x, y, ...)
```

Luacheck (1.2.0)

[warning] unused variable length argument

(W212)